gh_actions_exporter/cost.py

```diff
@@ -0,0 +1,18 @@
+from githubkit.webhooks import WorkflowJob
+
+from gh_actions_exporter.config import Settings
+
+
+class Cost(object):
+    def __init__(self, settings: Settings):
+        self.settings: Settings = settings
+
+    def get_job_cost(self, job_request: WorkflowJob, flavor_label: str) -> float:
```

Comment:
This function now can be implemented by the metrics class as is. Or at least we should, we can't leave duplicated methods if the logic already exists in the code as I'm sure we'll forget about it otherwise.